AsyncDisplayKit/ASTextNode.mm

```diff
@@ -1044,6 +1049,7 @@ - (void)setShadowColor:(CGColorRef)shadowColor
       CGColorRetain(shadowColor);
     }
     _shadowColor = shadowColor;
```

Comment:
Since you're already in this code, could you add `if (_shadowColor != NULL) { CGColorRelease(_shadowColor); }` so we stop leaking them 🚰